### ISSUE TITLE
Category GUI broken in latest version

### ISSUE THREAD

**Issue by Nuckerr** [pustinek/ItemFilter#11]
It seems that 359f678 broke/removed the category gui feature. Is there anyway to reenable that or am I missing something. (when I run `/itemfilter menu` it just doesnt give me a category menu)

**Comment by pustinek** [comment]
Yeah with version 2 I removed the category GUI since I found that simply allowing players to set items they want to filter is more intuitive, easier to manage and more powerful. 
With this version you don't have to worry about configuring new categories every time, so it's basically install and forget.
I've been using this version on my server for the last 2 years, and haven't had any issues and I'm not going back to the old system :smile: 

But if you want to use categories feel free to download it from spigot or clone it and reset to the commit before version 2

**Comment by andrewkm** [comment]
@Nuckerr @pustinek are you guys using this on 1.20.6 btw?
We did some initial tests and the plugin seems to work on 1.20.6, but wondering if there's any potential oversights in my testing and broken areas, since 1.20.5/6 was such a big change by Mojang.

**Comment by andrewkm** [comment]
@pustinek would you be able to also opt out of remapping on paper?
Pretty sure it would go under the shadowJar task in build.gradle.kts
```
        manifest {
            attributes("paperweight-mappings-namespace" to "mojang")
        }
```

**Comment by andrewkm** [comment]
Category GUI's were basically the most loved feature on our server. RIP.

**Comment by GG-MD** [comment]
> It seems that [359f678](https://github.com/pustinek/ItemFilter/commit/359f6781adf9168cc22819f23ea2d122d1292e9f) broke/removed the category gui feature. Is there anyway to reenable that or am I missing something. (when I run `/itemfilter menu` it just doesnt give me a category menu)

Write here if you still need this.

**Comment by andrewkm** [comment]
> > It seems that [359f678](https://github.com/pustinek/ItemFilter/commit/359f6781adf9168cc22819f23ea2d122d1292e9f) broke/removed the category gui feature. Is there anyway to reenable that or am I missing something. (when I run `/itemfilter menu` it just doesnt give me a category menu)
> 
> Write here if you still need this.
> 

Would be nice.

**Comment by GG-MD** [comment]
[ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)

> Menu categories, list your items on filtering. ;)

**Comment by andrewkm** [comment]
> [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> 
> > Menu categories, list your items on filtering. ;)

Submit a PR. Not a zip file.

**Comment by GG-MD** [comment]
> > [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> > > Menu categories, list your items on filtering. ;)
> 
> Submit a PR. Not a zip file.

I'm lazy.

**Comment by andrewkm** [comment]
> > > [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> > > > Menu categories, list your items on filtering. ;)
> > 
> > Submit a PR. Not a zip file.
> 
> I'm lazy.

Then kick rocks instead of distributing random zip files.

**Comment by GG-MD** [comment]
> > > > [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> > > > > Menu categories, list your items on filtering. ;)
> > > 
> > > Submit a PR. Not a zip file.
> > 
> > I'm lazy.
> 
> Then kick rocks instead of distributing random zip files.

This is a modified version — you should’ve built the project, tested the plugin in action, and then apologized for your words, considering how valuable what you were given is.
If it looks like the original ItemFilter, then go ahead and test it yourself.

**Comment by andrewkm** [comment]
> > > > > [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> > > > > > Menu categories, list your items on filtering. ;)
> > > > 
> > > > Submit a PR. Not a zip file.
> > > 
> > > I'm lazy.
> > 
> > Then kick rocks instead of distributing random zip files.
> 
> This is a modified version — you should’ve built the project, tested the plugin in action, and then apologized for your words, considering how valuable what you were given is.
> If it looks like the original ItemFilter, then go ahead and test it yourself.

Bro thinks his shits valuable lmao.

**Comment by GG-MD** [comment]
> > > > > > [ItemFilter-main.zip](https://github.com/user-attachments/files/20984101/ItemFilter-main.zip)
> > > > > > > Menu categories, list your items on filtering. ;)
> > > > > 
> > > > > Submit a PR. Not a zip file.
> > > > 
> > > > I'm lazy.
> > > 
> > > Then kick rocks instead of distributing random zip files.
> > 
> > This is a modified version — you should’ve built the project, tested the plugin in action, and then apologized for your words, considering how valuable what you were given is.
> > If it looks like the original ItemFilter, then go ahead and test it yourself.
> 
> Bro thinks his shits valuable lmao. 

Category list 😀![image](https://github.com/user-attachments/assets/9299d37a-fdf9-4c23-b0c0-300108952ec9)

Configured items that are currently not being picked up. 🤣![image](https://github.com/user-attachments/assets/075b98de-4754-48e4-ba9b-0424dcf68b2f)
